functions/src/modules/pm/categories/prizes/getPrize.ts

```diff
@@ -1,5 +1,48 @@
-import { ExpressFunction } from "../../../../@types";
+import type { ExpressFunction } from "../../../../@types";
+import { firebaseApp } from "../../../../config/firebaseConfig";
+import { expressErrorHandlerFactory } from "../../../../helpers";
+import { uasPermissionSwitch } from "../../../../systems/uas";
 
-const replaceMe: ExpressFunction = (req, res, next) => {};
+const getPrizeId: ExpressFunction = (req, res, next) => {
+    uasPermissionSwitch({
+      organizer: { accepted: execute },
+    })(req, res, next);
+  };
 
-export default replaceMe;
\ No newline at end of file
+
+
+const execute: ExpressFunction = (req, res, next) => {
+      const errorHandler = expressErrorHandlerFactory(req, res, next);
+      firebaseApp
+      .firestore()
+      .collection("Prize")
```

Comment:
This needs to be nested, so I think that it will end up as follows:
```suggestion
    .collection("prizeCategories)
    .doc(req.params.categoryId)
    .collection("prizes")
```